backend/libraries/group_community_common/src/user_cache.rs

```diff
@@ -17,15 +17,24 @@ pub struct CachedUser {
     pub diamond_membership_expires_at: Option<TimestampMillis>,
     #[serde(rename = "u", alias = "is_unique_person", default, skip_serializing_if = "is_default")]
     pub is_unique_person: bool,
+    #[serde(rename = "c", alias = "chit_earned", default, skip_serializing_if = "is_default")]
```

Comment:
No need for this alias, and we can remove the other ones, they were just needed when the shorter names were first introduced.